lib/internal/util.js

```diff
@@ -278,3 +278,14 @@ promisify.custom = kCustomPromisifiedSymbol;
 
 exports.promisify = promisify;
 exports.customPromisifyArgs = kCustomPromisifyArgsSymbol;
+
+exports.intrinsic = {
+  FunctionApply: Function.prototype.apply,
```

Comment:
Sadly I just realized that `util.promisify` did not follow the `module.exports = {}` pattern and should be updated. For new exports from this module, can you use the `module.exports = {}` above rather than the `exports.whatever = ...` model?